pom.xml

```diff
@@ -92,4 +157,14 @@
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    <build><plugins>
```

Comment:
No newline before this one, but should be fine and can be fixed later.

Reply:
+1